server/index.js

```diff
@@ -20,6 +20,7 @@ const proxyOptions = {
     proxyReq.setHeader("customer-id", process.env.customer_id);
     proxyReq.setHeader("x-api-key", process.env.api_key);
     proxyReq.setHeader("grpc-timeout", "60S");
+    proxyReq.setHeader("X-Source", "vectara-answer");
```

Comment:
Headers are case-insensitive, but we're lower-casing them elsewhere in this file so maybe change this to `x-source` for consistency.